unimport/console.py

```diff
@@ -1,11 +1,63 @@
+import argparse
 import os
-import sys
+import pathlib
 import tokenize
 
+from unimport.config import Config
 from unimport.files import get_files
 from unimport.unused import get_unused
 
 
+class Cli(object):
+    def parse_args(self):
+        parser = argparse.ArgumentParser(
+            description="Detect or remove unused Python imports."
+        )
+        parser.add_argument(
+            "source",
+            default=".",
+            nargs="?",
+            help="include file or folder to find the unused imports",
+        )
+        parser.add_argument(
+            "-c",
+            "--config",
+            help="read configuration from PATH",
+            metavar="PATH",
+            type=argparse.FileType("rb"),
+        )
+        parser.add_argument(
+            "-w",
+            "--write",
+            action="store_true",
+            help="remove unused imports automatically",
+        )
+        return parser.parse_args()
+
+    def run(self):
+        args = self.parse_args()
+        src = pathlib.Path(args.source)
+
+        if args.config:
+            config_file = pathlib.Path(args.config.name)
+        else:
+            config_file = None
+        config = Config(config_file=config_file)
+        py_files = get_files(src, config=config)
+        action_method = "write" if args.write is True else "print"
```

Comment:
This doesnt look future proof, I think if we add fixing we are going to need a third action to show the diff so this can be adjusted to a multiline if/elif.

Reply:
@isidentical Indeed **write** and **print** are opposite actions of each other one. but now I think that that can be possible to use both actions at the same time. I didn't add this feature yet but we can think about it later.

Now I convert it to a classical way using if-else statement.